rpcserverhandlers_test.go

```diff
@@ -6,28 +6,829 @@ package main
 
 import (
 	"errors"
+	"fmt"
+	"math"
+	"reflect"
 	"testing"
 
+	"github.com/decred/dcrd/blockchain/stake/v3"
+	"github.com/decred/dcrd/blockchain/v3"
+	"github.com/decred/dcrd/chaincfg/chainhash"
 	"github.com/decred/dcrd/chaincfg/v3"
 	"github.com/decred/dcrd/dcrjson/v3"
 	"github.com/decred/dcrd/dcrutil/v3"
+	"github.com/decred/dcrd/internal/rpcserver"
 	"github.com/decred/dcrd/rpc/jsonrpc/types/v2"
 )
 
-// testCfg holds the cfg for the test server.
-var testCfg = rpcserverConfig{ChainParams: chaincfg.MainNetParams()}
+// testRPCUtxoEntry provides a mock utxo entry by implementing the UtxoEntry interface
+type testRPCUtxoEntry struct {
+	rpcUtxoEntry
```

Comment:
Right, that is what I figured was the case.  I'm not overly concerned with it the way it is if you prefer to leave it, since this is test code, but I did want to point out that it is a bit loose in terms of correctness and will result in panics if new handlers are added without overriding the relevant methods.

That said, a panic versus an explicit "not implement yet" error in the tests really doesn't make much difference when it comes to test failure since it's a failure either way.  It might be a little easier on someone less familiar with the code trying to implement tests for a new handler to see a "Foo is not implemented yet" test failure as opposed to a panic, but it should be pretty easy to figure out either way.